03-Networking/README.md

```diff
@@ -26,6 +26,14 @@ At the end of the exercise, you will have a VPC network with 1 subnet and 1 Clou
 * [terraform-google-cloud-nat](https://github.com/terraform-google-modules/terraform-google-vpc)
 * [Cloud NAT](https://cloud.google.com/nat/docs/overview)
 
+## Preparation: Enable APIs
+When using any new service for the first time in a GCP project, you'll need to enable the APIs that you'll be using.
+
+Enable the [Compute API](https://cloud.google.com/compute/docs/reference/rest/v1/) using:
+```
+gcloud services enable compute.googleapis.com
```

Comment:
Wouldn't this be a good step to demo `google_project_services` or `terraform-google-modules/project-factory/google//modules/project_services`?